template/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.replikativ</groupId>
+    <artifactId>incognito</artifactId>
```

Comment:
The descriptions and IDs in here are still for incognito. Please adjust to konserve.

Reply:
Sorry, I didn't look at this file. I thought because it is a template, it is general for all libraries. I will adjust this